app/assets/js/rtMedia.backbone.js

```diff
@@ -954,6 +954,17 @@ jQuery( document ).ready( function( $ ) {
 			$( '#rtmedia-whts-new-upload-container > div' ).css( 'top', '0' );
 			$( '#rtmedia-whts-new-upload-container > div' ).css( 'left', '0' );
 
+			/**
+			 * NOTE: Do not change.
+			 * ISSUE: BuddyPress activity upload issue with Microsoft Edge
+			 * GL: 132 [ http://git.rtcamp.com/rtmedia/rtMedia/issues/132 ]
+			 * Reason: Trigger event not working for hidden element in Microsoft Edge browser
+			 * Condition to check current browser.
+			 */
+			if ( /Edge/.test( navigator.userAgent ) ) {
+				jQuery( this ).closest( '.rtm-upload-button-wrapper' ).find( 'input[type=file]' ).click();
```

Comment:
@BhargavBhandari90 Looks like this code runs on document ready on Edge, regardless if its clicked or not. 
This code should run on Button click event. Please check. @chandrapatel @milindmore22 
`jQuery( this ).closest( '.rtm-upload-button-wrapper' ).find( 'input[type=file]' ).click();`